superset/migrations/env.py

```diff
@@ -34,10 +34,13 @@
 fileConfig(config.config_file_name)
 logger = logging.getLogger("alembic.env")
 
-
-config.set_main_option("sqlalchemy.url", current_app.config["SQLALCHEMY_DATABASE_URI"])
+DATABASE_URI = current_app.config["SQLALCHEMY_DATABASE_URI"]
+if "sqlite" in DATABASE_URI:
+    logger.warning("SQLite DB support may not be supported by in future version")
```

Comment:
Grammar nit: could we use "SQLite Database support for metadata databases will be removed in a future version of Superset."